lib/msal-node/src/client/ManagedIdentityApplication.ts

```diff
@@ -134,6 +134,7 @@ export class ManagedIdentityApplication {
         }
 
         const managedIdentityRequest: ManagedIdentityRequest = {
+            disableInternalRetries: this.config.system.disableInternalRetries,
```

Comment:
Should this variable be defined per application and not per request? @bgavrilMS

Reply:
I think this variable should be defined for the ManagedIdentityApplication and not ManagedIdentityRequest.